Sources/SwiftlySearch/SwiftlySearch.swift

```diff
@@ -86,11 +109,25 @@ fileprivate struct SearchBar: UIViewControllerRepresentable {
             }
         }
         
+        var hidesSearchBarWhenScrolling: Bool = true {
+            didSet {
+                self.parent?.navigationItem.hidesSearchBarWhenScrolling = hidesSearchBarWhenScrolling
+            }
+        }
+        
         override func viewWillAppear(_ animated: Bool) {
-            self.parent?.navigationItem.searchController = searchController
+            setup()
         }
         override func viewDidAppear(_ animated: Bool) {
+            setup()
+        }
+        
+        private func setup() {
             self.parent?.navigationItem.searchController = searchController
+            self.parent?.navigationItem.hidesSearchBarWhenScrolling = hidesSearchBarWhenScrolling
+            
+            // make search bar appear at start (default behaviour since iOS 13)
+            self.parent?.navigationController?.navigationBar.sizeToFit()
```

Comment:
This does not seem to do anything that I could tell. Why was this added?

Reply:
Without this line of code, the search bar wouldn't show up when the view first appears and `hidesSearchBarWhenScrolling` is true. The default behavior (since iOS 13) in UIKit is to show the search bar initially, only hiding it when the user interacts with the scroll view. But sometimes there's a bug in UIKit, which lets the search bar disappear initially. So this is basically a bug fix until Apple fixes this bug.

Reply:
Alright, that's what I figured it was supposed to do, but I was unable to reproduce that for some reason (namely it hid itself by default even with that line).